src/js/background/badge.js

```diff
@@ -10,16 +10,29 @@ const badge = {
     browser.browserAction.setTitle({ tabId, title: "Containers disabled in Private Browsing Mode" });
   },
 
-  async displayBrowserActionBadge() {
+  async displayBrowserActionBadge(action) { 
     const extensionInfo = await backgroundLogic.getExtensionInfo();
-    const storage = await browser.storage.local.get({browserActionBadgesClicked: []});
-
-    if (MAJOR_VERSIONS.indexOf(extensionInfo.version) > -1 &&
-        storage.browserActionBadgesClicked.indexOf(extensionInfo.version) < 0) {
-      browser.browserAction.setBadgeBackgroundColor({color: "rgba(0,217,0,255)"});
-      browser.browserAction.setBadgeText({text: "NEW"});
+    function changeBadgeColorText(color, text){
+      browser.browserAction.setBadgeBackgroundColor({color: color});
+      browser.browserAction.setBadgeText({text: text});
+    }
+    if(action==="remove") {    
```

Comment:
Please change this action to `showVersionIndicator` and then update the call on line 5 to use it.

Reply:
thanks for the feedback. Please tell me what is meant by `showVersionIndicator` do you mean I should change the parameter `action` to `showVersionIndicator`?

Does the showversionindicator: get the version of the browser tab (in line 5 that will be incognito and in the messageHandler.js, that will be the 100th container tab)?

Reply:
Here, change it to `action==="showVersionIndicator"` and on line 5, change the call to pass `"showVersionIndicator"`

Reply:
thank you, I have effected the changes